packages/core/src/render3/instructions.ts

```diff
@@ -404,6 +406,7 @@ export function renderEmbeddedTemplate<T>(viewToRender: LView, tView: TView, con
       setPreviousOrParentTNode(null !);
 
       oldView = enterView(viewToRender, viewToRender[T_HOST]);
+      viewToRender[FLAGS_MORE] = 0;
```

Comment:
I think these flag resets would be less mysterious if they were wrapped in a descriptive function name, like

```suggestion
      resetPreOrderHookFlags();
```